enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -334,3 +334,24 @@
           clojure.lang.ExceptionInfo
           #"Sorry, but you'll need a test account to view this page. Please contact your administrator."
           (#'mt.jwt/fetch-or-create-user! "Test" "User" "test1234@metabase.com" nil)))))))
+
+(deftest get-jwt-token-test
+  (testing "should return a session token when token=true"
+    (with-jwt-default-setup
+      (let [jwt-iat-time (buddy-util/now)
+            jwt-exp-time (+ (buddy-util/now) 3600)
+            result       (saml-test/client-full-response :get 200 "/auth/sso"
+                                                         :token true
+                                                         :jwt (jwt/sign {:email
+                                                                         "rasta@metabase.com"
+                                                                         :first_name "Rasta"
+                                                                         :last_name  "Toucan"
+                                                                         :extra      "keypairs"
```

Comment:
did you want to test that extra keypairs come back too?

Reply:
Now that I think about it I guess it doesn't even matter if extra keypairs roundtrip; it's just about making sure they don't mess up the stuff we do care about

Reply:
Exactly - we could definitely fire those keypairs into the sun but I just want to make sure there isn't any extra weirdness when they're introduced